docs/migration_to_1.0.0.md

```diff
@@ -0,0 +1,72 @@
+# Migrating from Wasmer 0.x to Wasmer 1.0.0
```

Comment:
Live doc link:

https://github.com/wasmerio/wasmer/blob/docs/migration-doc/docs/migration_to_1.0.0.md

Reply:
I linked to it at the top!

Reply:
Didn't saw it! Resolving this conversation then!